src/common/modals/compositions/DepositModal/MainnetDepositModalContent.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="mainnet-deposit-modal-content">
+    <p
+      v-if="currentStep !== STEPS.chooseAssetStep"
+      class="mainnet-deposit-modal-content__lbl"
+    >
+      <span
+        :class="
+          [
+            'mainnet-deposit-modal-content__lbl',
+            'mainnet-deposit-modal-content__lbl--highlighted',
+          ].join(' ')
```

Comment:
In order to ease working with multiple classes you can create a small helper function that makes it look cleaner

```ts
export const mergeClasses = (...classes: (string | undefined)[]) => classes.join(' ')

mergeClasses('mainnet-deposit-modal-content__lbl', 'mainnet-deposit-modal-content__lbl--highlighted')
```

Reply:
done